common/app/common/configuration.scala

```diff
@@ -552,8 +552,6 @@ class GuardianConfiguration extends Logging {
     def mandatoryCredentials: AWSCredentialsProvider = credentials.getOrElse(throw new BadConfigurationException("AWS credentials are not configured"))
     val credentials: Option[AWSCredentialsProvider] = {
       val provider = new AWSCredentialsProviderChain(
-        new EnvironmentVariableCredentialsProvider(),
-        new SystemPropertiesCredentialsProvider(),
```

Comment:
How about switching the order instead of removing?  Do we want to stop people from using env variables? (Answer might be yes 😃 )